puppet/modules/simpsetup/manifests/ldap.pp

```diff
@@ -14,7 +14,7 @@
 ) {
 
   $_domain = split($domain, '\.')
-  $basedn = "dc=${ $_domain.join(',dc=')}"
+  $basedn = "dc=${ _domain.join(',dc=')}"
```

Comment:
Should use simp_options::ldap::base_dn.

Reply:
@lnemsick-simp it probably shouldn't use `simp_options::` here because of the way it's called, but now I look at it, this code is broken.  Latest commits fix this (and more importantly, add basic tests).